native/packages/react-native-bpk-component-banner-alert/readme.md

```diff
@@ -63,16 +63,17 @@ export default class App extends Component {
           style={styles.bannerAlert}
           type={ALERT_TYPES.WARN}
           message={translationHelper.translate('WARN_MESSAGE')}
-          actionButtonLabel="Dismiss"
-          onAction={this.onDismissablePress}
-          dismissable={this.state.showDismissable}
+          dismissButtonLabel="Dismiss"
```

Comment:
In some places we've got the fake `translationHelper` calls, and in others plain English. I reckon we should just stick to one of the other.